src/App.jsx

```diff
@@ -2,19 +2,58 @@ import logo from "/logo.png";
 import "./App.css";
 import { makeShuffledDeck } from "./utils.jsx";
 import { useState } from "react";
+import Button from "react-bootstrap/Button";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+// import { RowEntry } from "./components/RowEntry";
 
 function App(props) {
-  // Set default value of card deck to new shuffled deck
+  // Set default value of card deck to new shuffled deck (or in other words, we start with a shuffled deck of cards)
   const [cardDeck, setCardDeck] = useState(makeShuffledDeck());
-  // currCards holds the cards from the current round
+  // currCards holds the cards from the current round.
   const [currCards, setCurrCards] = useState([]);
+  // Set default round of game as false (i.e. not started).
+  const [gameStart, setgameStart] = useState(false);
+  // Set number of rounds won by Player 1 as 0 by default.
+  const [p1RoundsWon, setp1RoundsWon] = useState(0);
+  // Set number of rounds won by Player 2 as 0 by default.
+  const [p2RoundsWon, setp2RoundsWon] = useState(0);
 
+  const [isDisabled, setIsDisabled] = useState(false); // initial value is true
```

Comment:
What is disabled? Also, the initial value here is false, so the comment is wrong :)